website/docs/guides/text-search.mdx

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 4
+---
+
+# Full text search
+
+import CodeBlock from '@theme/CodeBlock';
+import MyDataSet from '!!raw-loader!./requests/1-insert-text-data.js';
+import CreateIndex from '!!raw-loader!./requests/2-create-text-index.js';
+import SearchQuery from '!!raw-loader!./requests/3-basic-text-search.js';
+import CompoundIndex from '!!raw-loader!./requests/4-compound-text-index.js';
+import RelevanceScore from '!!raw-loader!./requests/5-relevance-score.js';
+import TextSearchResponse from '!!raw-loader!./responses/3-basic-text-search.js';
+import RelevanceScoreResponse from '!!raw-loader!./responses/5-relevance-score.js';
+
+Full-text search is a technique for searching words or phrases across a large set of textual data.
+Unlike traditional queries that require exact matches, full-text search tries to understand what you’re searching for and bring up the best results.
+It is widely used in applications like search engines, e-commerce platforms, documentation searches, and content management systems.
+
+## Understanding text indexes
+
+A full-text index is fundamentally different from a regular database index.
+Instead of simply mapping fields to values, it:
+
+- Tokenizes text (splits them into words or phrases).
+- Removes stop words (such as "is", "the", "and").
+- Applies stemming (so "running" and "run" are treated as the same).
+- Stores word positions (so phrase searches and proximity queries work).
```

Comment:
What can I read to understand what `Stores word positions` is? Reading about phrase search and proximity query didn't help :see_no_evil: